zubhub_frontend/zubhub/src/store/actions/userActions.js

```diff
@@ -357,6 +357,9 @@ export const addComment = args => {
   };
 };
 
+export const viewNotification = props => {
+  return API.viewNotification(props)
+};
```

Comment:
Since this doesn't relate to the Redux state , we can simply call ```API.viewNotification()``` from the component itself and not include it here.